packages/server/src/utils/index.ts

```diff
@@ -325,7 +325,10 @@ export const getVariableValue = (paramValue: string, reactFlowNodes: IReactFlowN
  * @returns {INodeData}
  */
 export const resolveVariables = (reactFlowNodeData: INodeData, reactFlowNodes: IReactFlowNode[], question: string): INodeData => {
-    const flowNodeData = cloneDeep(reactFlowNodeData)
+    //skipping cloneDeep of instances
+    //as cloning them gives "Illegal invocation" Exception
+    const flowNodeDataWithoutInstance = cloneDeep(omit(reactFlowNodeData, ['instance']))
+    const flowNodeData = merge(flowNodeDataWithoutInstance, { instance: reactFlowNodeData.instance })
```

Comment:
This should be ok as instances are created only during init phase. your toughts @HenryHengZJ ?

Reply:
yeah tested and it works fine

Reply:
somehow when I tried the branch locally, this caused streaming stopped working:
![image](https://github.com/FlowiseAI/Flowise/assets/26460777/d9ef70ab-16a4-4afc-a692-cbc62e91a8aa)

need to look into this

Reply:
@HenryHengZJ In local with this code are you still getting Illegal Invocation Exception ?

Reply:
yes. I think i'll make another PR on top of this as I'll have to disable streaming when FaiSS is being used.